save.go

```diff
@@ -158,35 +158,63 @@ func getAllDeps(importPath string, cmds []string) []string {
 		buildContext.CgoEnabled = true
 		buildContext.UseAllFiles = useAllFiles
 
-		forward, _, errs := importgraph.Build(&buildContext)
-		for pkg, err := range errs {
-			// Lots of errors because of UseAllFiles.
-			if !useAllFiles {
-				log.Printf("error loading package %s: %v", pkg, err)
+		// Add the subpackages.
+		for path := range buildutil.ExpandPatterns(&buildContext, []string{subpackagePrefix + "..."}) {
+			_, err := buildContext.Import(path, "", 0)
+			if _, ok := err.(*build.NoGoError); ok {
+				continue
 			}
+			roots[path] = struct{}{}
```

Comment:
Shouldn't we bail if we get an error that's not `NoGoError` (or at least print it)?